src/PlatformClient.php

```diff
@@ -241,18 +244,20 @@ protected function cleanRequest(array $request)
      *   similar code to wait for the subscription's project to be provisioned
      *   and activated.
      */
-    public function createSubscription($region, $plan = 'development', $title = null, $storage = null, $environments = null, array $activationCallback = null)
+    public function createSubscription($catalog, $region, $plan = 'development', $title = null, $storage = null, $environments = null, array $activationCallback = null)
```

Comment:
Push the catalog to the back of the argument list -- for the backward compatibility reasons, and because the catalog must not be mandatory.